dev-aws/kafka-shared-msk/data-infra/connectors.tf

```diff
@@ -0,0 +1,124 @@
+resource "kafka_topic" "events_send" {
+  name               = "data-infra.uw.data-infra.product.v1.eventsend"
+  replication_factor = 3
+  partitions         = 15
+  config = {
+    "remote.storage.enable" = "true"
+    # infinite retention
+    "retention.ms" = "-1"
+    # keep data in hot storage for 1 day
+    "local.retention.ms" = "86400000"
+    # allow max 1 MB for a message
+    "max.message.bytes" = "1048576"
+    "compression.type"  = "zstd"
+    "cleanup.policy"    = "delete"
+  }
+}
+
+resource "kafka_topic" "dlq_requeue" {
+  name               = "data-infra.product.v1.events.requeue"
+  replication_factor = 3
+  partitions         = 1
+  config = {
+    # this is a test, and we need minimum and non-durable resources
+    "remote.storage.enable" = "false"
+    # 5 days
+    "retention.ms" = "432000000"
+    # allow max 1 MB for a message
+    "max.message.bytes" = "1048576"
+    "compression.type"  = "zstd"
+    "cleanup.policy"    = "delete"
+  }
+}
+
+resource "kafka_topic" "dlq" {
+  name               = "data-infra.product.v1.events.dlq"
+  replication_factor = 3
+  partitions         = 1
+  config = {
+    # this is a test, and we need minimum and non-durable resources
+    "remote.storage.enable" = "false"
+    # 5 days
+    "retention.ms" = "432000000"
+    # allow max 1 MB for a message
```

Comment:
Ok, I'll change it to 31 days although that feels a bit long - I've not configured remote storage because I expect the amount of data on these topics to be tiny